packages/grid/x-data-grid-pro/src/hooks/features/export/useGridExcelExport.tsx

```diff
@@ -0,0 +1,114 @@
+import * as React from 'react';
+import { useGridApiMethod, useGridLogger, GridExportDisplayOptions } from '@mui/x-data-grid';
+import { useGridRegisterPreProcessor, buildWarning, exportAs } from '@mui/x-data-grid/internals';
+import { DataGridProProps } from '../../../models/dataGridProProps';
+import { GridApiPro } from '../../../models/gridApiPro';
+import {
+  GridExcelExportApi,
+  GridExportExtension,
+  GridExcelExportOptions,
+} from './gridExcelExportInterface';
+import { buildExcel } from './serializer/excelSerializer';
+import { defaultGetRowsToExport, getColumns } from './serializer/utils';
```

Comment:
Since you created helpers does it make sence to also use them for the CSV export?

Reply:
Yes, I forget the PR was merged 👍